modules/jdbc-test/src/test/java/org/testcontainers/junit/SimpleDb2Test.java

```diff
@@ -0,0 +1,39 @@
+package org.testcontainers.junit;
+
+import com.zaxxer.hikari.HikariConfig;
+import com.zaxxer.hikari.HikariDataSource;
+import org.junit.Rule;
+import org.junit.Test;
+import org.testcontainers.containers.Db2Container;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+
+import static org.rnorth.visibleassertions.VisibleAssertions.assertEquals;
+
+
+public class SimpleDb2Test {
```

Comment:
isn't it already covered by the jdbc-test module?

Reply:
this is in the jdbc-test module. What test class/method is this duplicated by?

Reply:
my bad, please ignore this comment :)